classes/Score.py

```diff
@@ -9,7 +9,7 @@ def __init__(self):
 	def display(self,display_surface,WINDOW_WIDTH,WINDOW_HEIGHT,scoreCount):
 		# exercise: recreate the original display_score function inside of a class
 		# actually call it in the game loop
-		score_text = f'Score: {scoreCount}'
+		score_text = f'Score: {pygame.time.get_ticks() // 100}'
```

Comment:
I understand that you are trying to increment the score using the Pygame library function. But, it would be good if you can try implementing a proper logic for incrementing the score rather than using the library function. Let us keep this implementation for sprint2 as it is satisfying the MVP. But let's better try implementing the logic for the next sprint.